python/lib/communication/dmod/communication/maas_request/distribution.py

```diff
@@ -3,67 +3,27 @@
 from typing import Literal
 
 
-class DistributionBounds(Serializable):
+class Distribution(Serializable):
+    """
+    Represents the definition of a distribution of numbers
+    """
+
     minimum: int = 0
+    """The lower bound for the distribution"""
+
     maximum: int = 0
+    """The upper bound for the distribution"""
+
     distribution_type: Literal["normal"] = "normal"
+    """The type of the distribution"""
 
-    class Config:
+    class Config(Serializable.Config):
         feilds = {
```

Comment:
Typo or can me not raed?

Reply:
@christophertubbs, what typo? `Config` inheriting from `Serializable.Config`?

Reply:
what a blunder... thanks for catching this.